src/Standards/AcquiaDrupal/ruleset.xml

```diff
@@ -32,22 +32,4 @@
   <!-- Drupal Practice sniffs -->
   <rule ref="DrupalPractice.Commenting.ExpectedException"/>
```

Comment:
It's kind of curious to have only and exactly one rule from a stricter standard. I know it was already here, but the seeming incongruity is really highlighted now. (It's the only case where we _tighten_ the requirements instead of _loosening_ them.) Should we remove it? Or otherwise add an explanatory comment so future users don't have to search the codebase to figure out what it does like I did? 🙂